src/App.tsx

```diff
@@ -1,35 +1,18 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import Sidebar from './components/sidebar'
+import { BrowserRouter} from 'react-router-dom'
+import styles from './app.module.css'
```

Comment:
Sort imports, you can do so using vscode's commander (Ctrl/Cmd + Shift + P) "Typescript: Sort Imports".